server/src/main/java/com/exacaster/lighter/application/ApplicationStatusHandler.java

```diff
@@ -85,13 +88,13 @@ private ApplicationState trackStatus(Application app, ApplicationInfo info) {
 
     private ApplicationState checkZombie(Application app) {
         LOG.info("No info for {}", app);
-        if (app.getContactedAt() != null && app.getContactedAt().isBefore(LocalDateTime.now().minusMinutes(30))) {
+        if (app.getContactedAt() != null && app.getContactedAt().isBefore(LocalDateTime.now().minusMinutes(conf.getLighterZombieInterval().toMinutes()))) {
```

Comment:
Why not just now().minus(conf.getLighterZombieinterval()) ?

Reply:
thanks